app/controllers/cart_items_controller.rb

```diff
@@ -10,4 +10,10 @@ def create
     redirect_to items_path
   end
 
+  def destroy
+    @cart.contents.delete_if { |key, value| key == params[:id] }
```

Comment:
Unused block argument - `value`. If it's necessary, use `_` or `_value` as an argument name to indicate that it won't be used.